pkg/statistics/handle/globalstats/global_stats_test.go

```diff
@@ -921,6 +921,7 @@ func TestGlobalIndexStatistics(t *testing.T) {
 }
 
 func TestIssues24349(t *testing.T) {
+	t.Skip()
```

Comment:
If we keep version 1 analyze, it is best to run version 1 test. Or just delete the case directly

Reply:
+1. Maybe we should update it or set the version to v1.

Reply:
There might be another problem, but I'm still trying not to skip or use v1 to test it.